unit-8/homework-unit8/menu/style/style.scss

```
@@ -0,0 +1,158 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,400;1,500;1,700;1,900&display=swap');
+
+$backgroundColor: #EEEEEE;
```

```suggestion
$background-color: #EEEEEE;
```
- Hãy viết tất cả bằng chữ thường
- Hãy xác định rõ `background` cho cái nào, hay đặc điểm của background đó, vd: `background-active-color`, `background-primary-color`, `background-highlight-color`, ...